spec/docker/timestamp_tag_spec.rb

```diff
@@ -0,0 +1,9 @@
+RSpec.describe Kuby::Docker::TimestampTag, ".try_parse" do
```

Comment:
Please add `require 'spec_helper'` at the top of this file. You should be able to use `describe` by itself here too, and create a new context for `.try_parse` i.e:

```ruby
require 'spec_helper'

describe Kuby::Docker::TimestampTag do
  context '.try_parse' do
    ...
  end
end
```